MatrixSDK/Contrib/Swift/Data/MXSessionEventListener.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MXSessionEventListener.swift
+//  MatrixSDK
+//
+//  Created by Avery Pierce on 7/12/17.
+//  Copyright © 2017 matrix.org. All rights reserved.
```

Comment:
Whoops! My mistake. I had fixed this in my local copy already, but I guess I forgot to push the change to Github before submitting the PR. It should be fixed now.